repository-data/webfiles/src/main/resources/site/freemarker/hee/catalog/publicationlandingpage-main.ftl

```diff
@@ -18,182 +18,170 @@
-    <div class="nhsuk-review-date" style="margin-top:0px">
-        <p class="nhsuk-body-s">
-            Published: ${getDefaultFormattedDate(publishedDate)}<br>
-            <#if updatedDate?has_content>Updated: ${getDefaultFormattedDate(updatedDate)}<br></#if>
-            ${fileType?upper_case}${(fileType = 'WEB')?then('',', ' + fileLengthInKB + 'kB')}
-        </p>
+    <span>Published: ${getDefaultFormattedDate(publishedDate)}</span>
+    <#if updatedDate?has_content><span>Updated: ${getDefaultFormattedDate(updatedDate)}</span></#if>
+    <span>${fileType?upper_case}${(fileType = 'WEB')?then('',', ' + fileLengthInKB + 'kB')}</span>
+
+</#macro>
+
+<#macro docIcon fileType>
+    <div class="hee-publication-doc__wrapper">
+        <div class="hee-publication-doc__icon">
+            <div class="hee-publication-doc__icon__page"></div>
+            <div class="hee-publication-doc__icon__corner"></div>
+        </div>
+        <div class="hee-publication-doc__icon__title">${fileType}</div>
+
     </div>
 </#macro>
 
 <#if document??>
-    <div class="nhsuk-width-container">
-        <main id="maincontent" role="main" class="nhsuk-main-wrapper">
+    <main id="maincontent" role="main" class="page page--rightbar">
+        <#--  Renders title & subtitle  -->
+        <div class="page__header">
             <div class="nhsuk-width-container">
-
-                <#--  Renders title & subtitle  -->
-                <div class="nhsuk-grid-row">
-                    <div class="nhsuk-grid-column-two-thirds">
-                        <h1>
-                            <span role="text">${document.title}
-                                <span class="nhsuk-caption-xl nhsuk-caption--bottom">
-                                    ${document.subtitle!}
-                                </span>
-                            </span>
-                        </h1>
-                    </div>
+                <h1>${document.title}</h1>
+                <span class="nhsuk-caption-xl">${document.subtitle!}</span>
+            </div>
+        </div>
+
+        <div class="nhsuk-width-container">
+            <div class="page__layout">
+                <div class="page__main">
+                    <#-- Author and published date -->
+                    <p class="nhsuk-body-s nhsuk-u-secondary-text-color">
+                        <@fmt.message key="publication.by" var="byLabel" />
+                        <@fmt.message key="published.on" var="publishedOnLabel"/>
+
+                        <#assign commaSeparatedAuthorNames>${getCommaSeparatedAuthorNames(document.authors)}</#assign>
+
+                        ${publishedOnLabel} ${document.publicationDate.time?datetime?string['dd MMMM yyyy']}<#if commaSeparatedAuthorNames?has_content>, ${byLabel} ${commaSeparatedAuthorNames}</#if>
+                    </p>
+                    <#-- End Author and published date -->
+
+                    <#--  Renders summary  -->
+                    <#if document.summary??>
```

Comment:
FE seems to be suggesting `<p class="nhsuk-body-l">...</p>` snippet to render summary
```
<p class="nhsuk-body-l">Nulla vitae est vel metus aliquam pharetra vel quis ipsum. Pellentesque sit amet nulla a nibh elementum interdum at vel mi.</p>
```
Could we impl. that ?